### ISSUE TITLE
Snippet Tabstop Coloring

### ISSUE THREAD

**Issue by macintacos** [ahmadawais/shades-of-purple-vscode#34]
Currently, when expanding a snippet, this theme does not provide a background for a tabstop. Consider the following snippet definition (this is a user-defined snippet):

```
	"Typinator_Variable_Definition": {
		"prefix": "typ-var",
		"scope": "markdown",
		"body": "{{${1:variable name}=$0}}↩",
		"description": "Typinator Variable Definition"
	}
```

When I open a Markdown type and use `ctrl+space` to bring up snippets I have available, I choose `typ-var` for exansion, like so:

![](https://cl.ly/bf6faac883fe/Image%202019-04-10%20at%2011.21.31%20PM.png)

Upon hitting "enter", this provides me with a snippet that has expanded to show the following:

![](https://cl.ly/ef8dfe55acc7/Image%202019-04-10%20at%2011.21.54%20PM.png)

the characters "variable name" are actually highlighted for editing immediately, but the theme actually shows only a _very subtle_ difference in background color change. This made it not-obvious that those characters were meant to be placeholders (which is the whole point of the snippet anyway).

To get around this, I made the following change in my `settings.json` file:

```
  "workbench.colorCustomizations": {
    "editor.snippetTabstopHighlightBackground": "#FC87CF3f"
  }
```

While this works around the issue for this simple use-case, I'm sure there are several other items that need theming as well, so I'd rather pass that kind of work on to the responsibility of the theme instead 😄would it be possible to implement colors for these snippet tokens? More information for the specific user-editable variables can be found here: [Theme Color > Snippets](https://code.visualstudio.com/api/references/theme-color#snippets)

**Comment by ahmadawais** [comment]
Looking into it ;)

**Comment by ahmadawais** [comment]
Check out the latest version of 🦄 Shades of Purple theme. Which includes the correct highlighting for tabStops :).


![tabstops highlighting](https://on.ahmda.ws/16980d/c)

**Comment by ahmadawais** [comment]
FIXED in 8cc25c8c91b6100564c519fe139825c4ed46f2dd
